Doc/library/asyncio-stream.rst

```diff
@@ -50,40 +52,74 @@ Stream functions
    <coroutine>`; if it is a coroutine function, it will be automatically
    converted into a :class:`Task`.
 
-   The rest of the arguments are all the usual arguments to
-   :meth:`~AbstractEventLoop.create_server()` except *protocol_factory*; most
-   common are positional *host* and *port*, with various optional keyword
-   arguments following.
+   When specified, the *loop* argument determines which event loop to use,
```

Comment:
On line 47, consider adding:
```
The callback for a connected client, *client_connected_cb*, contains a *client_reader* and *client_writer*. ... The *client_connected_cb* parameter may be a plain callback function or a :ref:`coroutine function <coroutine>`. If the parameter is a coroutine function, it will be automatically converted into a :class:`Task`.
```

Reply:
I reworded the whole paragraph for clarity.

Reply:
Nice. Thanks @elprans